admin/assets/javascripts/typeahead.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Typeahead component
+ */
+
+/* global $ */
+$(function () {
+  'use strict'
+  if (!window.GOVUK) {
+    window.GOVUK = {}
+  }
+  window.GOVUK.typeahead = {
```

Comment:
How do you feel about this naming convention?
Shall I rather change it to autoComplete to match gds naming?

Reply:
can do.  don't mind either way, they are both decent names

Reply:
i notice in createComponent below, you are using autocomplete as var names.  so maybe use that

Reply:
@GuyHarwood thought so as well for naming consistency, it's done.